packages/contracts/contracts/interfaces/v0.8.x/ISplitAtomicFactoryV0.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Creatd By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+import {Split} from "./ISplitAtomicV0.sol";
+
+interface ISplitAtomicFactoryV0 {
+    /**
+     * @notice This contract was deployed.
+     * @param implementation address with the implementation of the contract
+     * @param type_ type of this contract
+     */
+    event Deployed(address indexed implementation, bytes32 indexed type_);
+    /**
+     * @notice New split atomic contract was created.
+     * @param splitAtomic address of the newly created split atomic contract
+     */
+    event SplitAtomicCreated(address indexed splitAtomic);
+
+    /**
+     * @notice Initializes the contract with the provided `splits`.
+     * Only callable once.
+     * @param splits Splits to configure the contract with. Must add up to
+     * 10_000 BPS.
+     * @return splitAtomic The address of the newly created split atomic
+     * contract
+     */
+    function createSplit(
+        Split[] calldata splits
+    ) external returns (address splitAtomic);
```

Comment:
TODO - add a 22.22% constraint to AB Admin here

Reply:
added in [a2c9c2e](https://github.com/ArtBlocks/artblocks-contracts/pull/1268/commits/a2c9c2e556520326d4f329784f6e2845ad7d8951)